.github/integration-in-memory-tests.yml

```
@@ -443,7 +443,7 @@ runner-test-matrix:
     triggers:
       - PR Integration CCIP Tests
       - Nightly Integration CCIP Tests
-    test_cmd: cd smoke/ccip && go test ccip_aptos_messaging_test.go -run "Test_CCIP_Messaging_EVM2Aptos" -timeout 20m -test.parallel=1 -count=1 -json
+    test_cmd: cd smoke/ccip && go test -run "Test_CCIP_Messaging_EVM2Aptos" -timeout 20m -test.parallel=1 -count=1 -json
```

Alternatively, could drop the cd?
```suggestion
    test_cmd: go test ./smoke/ccip -run "Test_CCIP_Messaging_EVM2Aptos" -timeout 20m -test.parallel=1 -count=1 -json
```

---

Changing this in: https://github.com/smartcontractkit/chainlink/pull/19365